core/joker.lua

```diff
@@ -73,6 +72,9 @@ function SMODS.injectJokers()
             effect = joker.effect,
             cost = joker.cost,
             cost_mult = 1.0,
+            blueprint_compat = joker.blueprint_compat,
+            eternal_compat = joker.eternal_compat,
+            effect = joker.effect

```

Comment:
There's a missing comma on this line, which crash the game once the payload has been injected